src/common/vector/value_vector.cpp

```diff
@@ -104,13 +104,13 @@ void ValueVector::copyValue(uint8_t* dest, const Value& value) {
     } break;
     case LIST: {
```

Comment:
Don't we need to add branch for TENSOR (FIXED_LIST) too here?

Reply:
I think this function will only be used when we are trying to add a FIXED_LIST literal, however we don't support FIX_LIST literal.